app/src/main/java/com/cornellappdev/android/eatery/ui/components/details/FavoritesToggle.kt

```diff
@@ -0,0 +1,88 @@
+package com.cornellappdev.android.eatery.ui.components.details
+
+import androidx.compose.foundation.BorderStroke
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.cornellappdev.android.eatery.R
+import com.cornellappdev.android.eatery.ui.theme.EateryBlue
+import com.cornellappdev.android.eatery.ui.theme.EateryBlueTypography
+import com.cornellappdev.android.eatery.ui.theme.GrayZero
+import com.cornellappdev.android.eatery.ui.viewmodels.ToggleViewModel
+import com.cornellappdev.android.eatery.util.EateryPreview
+
+@Composable
+fun ActiveToggle(
+    onClick: () -> Unit,
+    label: String
+) {
```

Comment:
Hmm okay this is a bit sus, the existence of separate `Composable`s for `ActiveToggle` and `InactiveToggle`. The two functions are VERY similar, in that there's a good amount of code reuse with perhaps a few lines that are only slightly different.

Let's make this a `FavoritesToggle` function that takes in an `active: Boolean` parameter instead. If it's active, the correct colors will be set appropriately--if it's inactive, same thing.

That way we cut down on code. Makes it more readable and arguably easier to interface with--all you need to do is pass in `active = true` to the toggle that's supposed to be true.